go.mod

```diff
@@ -0,0 +1,10 @@
+module gopkg.in/ns1/ns1-go.v2
```

Comment:
When I run `go mod tidy` on a new clone of `master`, my `go.mod` only requires `github.com/stretchr/testify v1.4.0`.  Any idea why all those indirect requirements are in there or if they are needed?